sass/styles/main.scss

```diff
@@ -1497,3 +1497,46 @@ cq-share-dialog {
 .cq-menu-dropdown {
 	line-height: 2.1em;
 }
+
+
+
+
+
+$spot-color: #0298d3;
+
+.cq-marker-spot{
```

Comment:
I think just call it `cq-spot` would do. It's not a marker; it follows the current quote